packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -104,7 +114,146 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
+    actionButtonContent: 'Allow'
+  };
 
+  const cameraGrantedDropdown = (
+    <Dropdown
+      data-ui-id="call-composite-local-camera-settings"
+      aria-labelledby={'call-composite-local-camera-settings-label'}
+      placeholder={defaultPlaceHolder}
+      options={cameraPermissionGranted ? getDropDownList(props.cameras) : [{ key: 'deniedOrUnknown', text: '' }]}
+      styles={dropDownStyles(theme)}
+      disabled={!cameraPermissionGranted}
+      errorMessage={
+        props.cameraPermissionGranted === undefined || props.cameraPermissionGranted
+          ? undefined
+          : locale.strings.call.cameraPermissionDenied
+      }
+      defaultSelectedKey={
+        micPermissionGranted
+          ? props.selectedCamera
+            ? props.selectedCamera.id
+            : props.cameras
+            ? props.cameras[0]?.id
+            : ''
+          : 'deniedOrUnknown'
+      }
+      onChange={(event, option, index) => {
+        props.onSelectCamera(props.cameras[index ?? 0], localVideoViewOptions);
+      }}
+      onRenderTitle={(props?: IDropdownOption[]) => onRenderTitle('Camera', props)}
+    />
+  );
+  /* @conditional-compile-remove(call-readiness) */
+  const cameraBlockedDropdown = (
+    <_DevicePermissionDropdown
+      styles={dropDownStyles(theme)}
+      onClickActionButton={async () => {
+        await adapter.askDevicePermission({ video: true, audio: false });
+        if (cameraPermissionGranted) {
+          adapter.queryCameras();
+        }
+      }}
+      strings={devicePermissionDropdownStringsCamera}
+      icon={<Icon iconName="ControlButtonCameraOn" style={{ height: '1.25rem', marginRight: '0.625rem' }} />}
+    />
+  );
+
+  const microphoneGrantedDropdown = (
+    <>
+      {roleCanUseMic && (
+        <Dropdown
+          aria-labelledby={'call-composite-local-sound-settings-label'}
+          placeholder={defaultPlaceHolder}
+          styles={dropDownStyles(theme)}
+          disabled={!micPermissionGranted}
+          errorMessage={
+            props.microphonePermissionGranted === undefined || props.microphonePermissionGranted
+              ? undefined
+              : locale.strings.call.microphonePermissionDenied
+          }
+          options={micPermissionGranted ? getDropDownList(props.microphones) : [{ key: 'deniedOrUnknown', text: '' }]}
+          defaultSelectedKey={
+            micPermissionGranted
+              ? props.selectedMicrophone
+                ? props.selectedMicrophone.id
+                : defaultDeviceId(props.microphones)
+              : 'deniedOrUnknown'
+          }
+          onChange={(
+            event: React.FormEvent<HTMLDivElement>,
+            option?: IDropdownOption | undefined,
+            index?: number | undefined
+          ) => {
+            props.onSelectMicrophone(props.microphones[index ?? 0]);
+          }}
+          onRenderTitle={(props?: IDropdownOption[]) => onRenderTitle('Microphone', props)}
+        />
+      )}
+      <Dropdown
```

Comment:
Check with Juan on this but I think speakers should always be showing as this isn't dependant on having mic permissions

Reply:
Haven't got a reply from Juan yet but I believe you are right! Modify the code to not hide the speaker dropdown for now